--- mate-text-editor-1.6.1/configure.ac.orig	2013-12-31 17:57:30.000000000 +0100
+++ mate-text-editor-1.6.1/configure.ac	2014-01-05 13:26:46.085080477 +0100
@@ -424,11 +424,11 @@
 	AC_MSG_RESULT([$PYGOBJECT_DEFSDIR])
 
 	AC_MSG_CHECKING([for pygtk codegen])
-	PYGTK_CODEGEN="$PYTHON `$PKG_CONFIG --variable=codegendir pygtk-2.0`/codegen.py"
+	PYGTK_CODEGEN="$PYTHON `$PKG_CONFIG --variable=codegendir pygtk-2.0`/codegen.pyc"
 	AC_MSG_RESULT([$PYGTK_CODEGEN])
 
 	AC_MSG_CHECKING([for pygtk h2def])
-	PYGTK_H2DEF="$PYTHON `$PKG_CONFIG --variable=codegendir pygtk-2.0`/h2def.py"
+	PYGTK_H2DEF="$PYTHON `$PKG_CONFIG --variable=codegendir pygtk-2.0`/h2def.pyc"
 	AC_MSG_RESULT([$PYGTK_H2DEF])
 
 	AC_SUBST([PYGTK_DEFSDIR])
